packages/snaps-controllers/src/snaps/SnapController.ts

```diff
@@ -3626,6 +3626,10 @@ export class SnapController extends BaseController<
       const [jsonRpcError, handled] = unwrapError(error);
 
       if (!handled) {
+        logError(
+          `"${snapId}" threw an unhandled error and crashed:`,
```

Comment:
Not a huge fan of this error message, maybe something like
`${snapId} terminated: Unhandled error`

Reply:
What about 
```suggestion
          `"${snapId}" crashed due to an unhandled error:`,
```

Reply:
Better but I don't like the fact that the error message looks like a spoken sentence. NIT anyways, if you deem that this error message is sufficient let's go with it

Reply:
I guess we don't have any standard for error message when the snap is terminated, let's go with the latest one if you think it's enough

Reply:
Applied!